server/src/infra/graphql/query.rs

```diff
@@ -661,18 +662,18 @@ pub fn serialize_attribute(
 }
 
 impl<Handler: BackendHandler> AttributeValue<Handler> {
-    fn from_schema(a: DomainAttributeValue, schema: &DomainAttributeList) -> FieldResult<Self> {
-        match schema.get_attribute_schema(&a.name) {
-            Some(s) => Ok(AttributeValue::<Handler> {
+    fn from_schema(a: DomainAttributeValue, schema: &DomainAttributeList) -> Option<Self> {
+        schema
+            .get_attribute_schema(&a.name)
+            .map(|s| AttributeValue::<Handler> {
                 attribute: a,
                 schema: AttributeSchema::<Handler> {
                     schema: s.clone(),
                     _phantom: std::marker::PhantomData,
                 },
                 _phantom: std::marker::PhantomData,
-            }),
-            None => Err(FieldError::from(format!("Unknown attribute {}", &a.name))),
-        }
+            })
+        // None => Err(FieldError::from(format!("Unknown attribute {}", &a.name))),
```

Comment:
You want to just swallow errors? That seems risky.

Reply:
Fields that are not `is_visible` are not in the schema if we are not an admin. Not sure where that was stemming from, but IIRC it comes from the method that returns and filters the schema based on the authenticated user earlier, so it felt better to just ignore those fields here than to change the backend that deals with permissions and move the filtering to the graphql frontend.

...now that I'm writing that out, I'm thinking that maybe I can add some filtering to the part of the backend that lists the attributes of the user and leave this unchanged... :thinking:

Reply:
Wait, we return the _values_ for fields that are not in the schema because they're not visible?

Definitely broken... Can you file that as a bug?

Reply:
oh, we're still inside the server, it's graphql's query.rs that's supposed to do the filtering, WAI